bundles/core/src/test/resources/page/exporter-templated-page.json

```diff
@@ -15,9 +15,9 @@
             "xdm:template": "/conf/coretest/settings/wcm/templates/product-page",
             "repo:path": "/core/content/page/templated-page.html",
             "xdm:tags": [
-                "two",
                 "three",
-                "one"
+                "one",
```

Comment:
Why does the order change here?

Reply:
The tags are saved in an unordered structure. The order has changed with the updated uber-jar and should remains stable after the update.